src/loader/mod.rs

```diff
@@ -260,14 +274,24 @@ impl KernelLoader for Elf {
         kernel_image
             .seek(SeekFrom::Start(ehdr.e_phoff))
             .map_err(|_| Error::SeekProgramHeader)?;
-        let phdrs: Vec<elf::Elf64_Phdr> = unsafe {
-            // Reading the structs is safe for a slice of POD structs.
-            struct_util::read_struct_slice(kernel_image, ehdr.e_phnum as usize)
-                .map_err(|_| Error::ReadProgramHeader)?
-        };
+
+        // TODO: use `Bytes` trait methods when rust-vmm/vm-memory#83 is merged.
```

Comment:
It's merged but not yet released on crates.io.

Reply:
> It's merged but not yet released on crates.io.

Then, should we release a new version of vm-memory before this gets merged?
I don't mind with the current syntax, either. 
In case we stick to this syntax, the comment needs to be updated.

Reply:
Updated the PR with the newly available functionality. The comments are no longer needed and were removed.